machines/slurm.train-neural-lam.sh

```diff
@@ -1,15 +1,6 @@
 #!/bin/bash -l
-#SBATCH --job-name=HAS-NeuralLam
-#SBATCH --time=1-00:00:00
-#SBATCH --nodes=1
-#SBATCH --ntasks-per-node=8
-#SBATCH --gres=gpu:8  #per node
-#SBATCH --no-requeue
-#SBATCH --partition=prodq
-#SBATCH --exclusive
-#SBATCH --account=cu_0003
-#SBATCH --output=/dcai/users/schhau/git-repos/mllam-exps/logs/neurallam.%j.log
-#SBATCH --error=/dcai/users/schhau/git-repos/mllam-exps/logs/neurallam.%j.log
+#SBATCH --output=/dcai/users/${USER}/git-repos/mllam-exps/logs/neurallam.%j.log
+#SBATCH --error=/dcai/users/${USER}/git-repos/mllam-exps/logs/neurallam.%j.log
```

Comment:
```suggestion
#SBATCH --output=/dcai/users/%u/logs/neurallam.%j.log
#SBATCH --error=/dcai/users/%u/logs/neurallam.%j.log
```
as discussed, slurm doesn't parse environment variables, and bash just sees this as a comment, so we can use the slurm variable instead